frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-row.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ArrowCircleDownIcon } from '@patternfly/react-icons';
+import { LoadingInline, pluralize } from '@console/internal/components/utils';
+
+const pluralizeWithNotation = (i: number, suffix: string, singular: string) =>
+  `${i}${suffix} ${i === 1 ? singular : `${singular}s`}`;
+
+const formatCount = (count: number) => {
+  if (count < 1000) return pluralizeWithNotation(count, '', 'Object');
+  if (count >= 1000 && count < 1000000) return pluralizeWithNotation(count / 1000, 'K', 'Object');
+  return pluralizeWithNotation(count / 1000000, 'M', 'Object');
+};
+
+const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status }) => (
+  <div className="co-buckets-card__row-status-item">
+    {_.isNil(status) ? (
+      <span className="co-buckets-card__row-subtitle">Unavailable</span>
+    ) : Number(status) > 0 ? (
+      <React.Fragment>
+        <div>
+          <ArrowCircleDownIcon />
+        </div>
+        <div className="co-buckets-card__row-status-item-text">{status}</div>
+      </React.Fragment>
+    ) : null}
+  </div>
+));
+
+const BucketsRow: React.FC<BucketsRowProps> = React.memo(
+  ({ bucketsCount, title, objectsCount }) => {
+    let substring = 'Unavailable';
```

Comment:
could be ternary condition
```suggestion
    const substring = _.isNil(objectsCount) ? 'Unavailable' : formatCount(Number(objectsCount));
```

Reply:
can you rename the constant to `subtitle` to better reflect it's purpose ?